duckdb.go

```diff
@@ -15,8 +15,11 @@ import (
 	"github.com/marcboeker/go-duckdb/mapping"
 )
 
+var instanceCache mapping.InstanceCache
+
 func init() {
 	sql.Register("duckdb", Driver{})
+	instanceCache = mapping.CreateInstanceCache()
```

Comment:
It might be better to use `sync.Once` to lazily initialise the instance cache only when `NewConnector` is called rather than calling it in `init()`.

Reply:
That's a great suggestion, thanks! I will change it.